src/assets/stylesheets/components/_alert.scss

```
@@ -0,0 +1,56 @@
+.alert {
+  display: flex;
+  width: 100%;
+  margin-bottom: 2rem;
```

We prefer using `rem` unit, but we can add the px value here. There is a function to convert `px` to `rem` in `src/assets/stylesheets/functions/_sizing.scss`

```suggestion
  margin-bottom: rem(32px);
```

---

This is so helpful thanks 🙏